projects/plugins/jetpack/extensions/blocks/tiled-gallery/editor.native.scss

```diff
@@ -0,0 +1,3 @@
+.galleryAppender {
+	padding-top: $grid-unit-20;
+}
```

Comment:
This doesn't appear to be used (a selector with the same name is imported from `styles.native.scss`.

Reply:
@SiobhyB I removed this selector as it indeed seems like a dupe.
I left the empty file in place because otherwise [this import statement](https://github.com/Automattic/jetpack/blob/196470043fb5d91cd19c8adaa4e67c3918bc5d4c/projects/plugins/jetpack/extensions/blocks/tiled-gallery/index.js#L28) imports the web's SCSS file, which breaks mobile for a number of reasons (unsupported media queries, etc).
I don't know of a better way of doing this (conditional platform-specific imports? I don't think they exist), are you OK with leaving this empty file here?

Reply:
Leaving the file empty sounds good! 💯